crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -380,74 +350,8 @@ $env.config = {
             description_text: yellow
         }
       }
-      # Example of extra menus created using a nushell source
```

Comment:
If we're going to remove examples, which is fine, we need to stash them somewhere else like documentation. No one is going to come up with how menus are supposed to be created on their own. They're pretty complex.

Reply:
i add them back in the `nu_scripts` in https://github.com/nushell/nu_scripts/pull/550 :yum:

Reply:
my comment below may supersede what I said here, if these menus were also defaults.

Reply:
> my comment below may supersede what I said here, if these menus were also defaults.
>
mm from what i understand in the Rust implementation of `config.rs`, these three menus were not default, i.e. when running `nu --no-config-file`, you should not be able to use these :thinking: 
this is why i've removed them from `default_config.nu` and moved them to `nu_scripts` in nushell/nu_scripts#550

Reply:
ok, that's great. equally important is that whatever the default menus are, we need to somehow put them in here I think.

Reply:
yup, i think they are all there :crossed_fingers: